### PR TITLE
Add edit and delete template flex

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -506,6 +506,10 @@ open_orchestra_backoffice:
         delete:
             title: Souhaitez-vous supprimer ce gabarit ?
             text: Cette opération est définitive.
+    templates_flex:
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.fr.yml]: There's no english version ?

File: BackofficeBundle/Resources/translations/role.en.yml [hunk 2]
```diff
@@ -11,6 +11,10 @@ open_orchestra_role:
     role_access_create_template: Create a page template
     role_access_update_template: Update a page template
     role_access_delete_template: Delete a page template
+    role_access_tree_template_flex: Access page templates flex
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/role.en.yml]: There's no fr version ?

**Comment by itkg-alavieille** [comment]
@itkg-ngilain pr updated add translation

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**